src/textual_summary/tests/__snapshots__/generic_table_row.test.js.snap

```diff
@@ -1,90 +1,153 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Simple Table renders multi-value just fine... 1`] = `
-<tr
-  className=""
+<GenericTableRow
```

Comment:
Could you fix the tests? Looks like mount from enzyme adds default div and that's not good, best would be to change `generic_table_row.test.js` and add `<table><tbody><GenericTableRow ... /></tbody></table>`. And of course update snapshots. @martinpovolny should know about this since he wrote these tests.

Reply:
@karelhala done.